docs/docs/dev_docs/tutorials/token_portal/setup.md

```diff
@@ -228,12 +222,6 @@ Then create a jest config file: `jest.config.json`
 }
 ```
 
-You will also need to install some dependencies:
-
-```bash
-yarn add --dev typescript @types/jest ts-jest
```

Comment:
Is typescript no longer a dependency?

Reply:
it is, meant to move it to a different command. good catch